### DISCUSSION TITLE
Read consistency of One in cassandra

### DISCUSSION THREAD

**Discussion by t4nmoy** [seaweedfs/seaweedfs#2430] [Q&A]
Hello,

from this [code](https://github.com/chrislusf/seaweedfs/blob/c77c0de418fab5e7230a1611df18d3dfdeb3b747/weed/filer/cassandra/cassandra_store_kv.go#L27) it seems it is using [this](https://github.com/chrislusf/seaweedfs/blob/c77c0de418fab5e7230a1611df18d3dfdeb3b747/weed/filer/cassandra/cassandra_store.go#L55) consistency (LocalQuorum) for read also. is it possible to use read consistency of one  [here](https://github.com/chrislusf/seaweedfs/blob/c77c0de418fab5e7230a1611df18d3dfdeb3b747/weed/filer/cassandra/cassandra_store_kv.go#L27)

Thanks

**Comment by chrislusf** [comment]
The writes is using LocalQuorum, so need to use LocalQuorum in order to read the data back correctly.

If you need to change the behavior, you can send a PR. Should be easy to do.
